lib/pdf_fill/forms/va21p527ez.rb

```diff
@@ -728,13 +730,11 @@ def expand_financial_acct(recipient, financial_acct, financial_accts)
       end
 
       def zero_financial_accts(financial_accts)
+        multi_line_accts = %w[socialSecurity salary]
         financial_accts.each do |acct_type, accts|
-          if accts.size.zero? && acct_type != 'additionalSources'
-            accts << {
-              'recipient' => 'Myself',
-              'amount' => 0
-            }
-          end
+          accts << DEFAULT_FINANCIAL_ACCT if accts.size.zero?
+          pad_accts = multi_line_accts.include?(acct_type) && accts.size < 2
+          (2 - accts.size).times { |_| accts << DEFAULT_FINANCIAL_ACCT } if pad_accts
```

Comment:
Can a social security or salary acct type have MORE than two lines?

Reply:
If Im reading this correctly then you're just trying to pad a default account unless its either socialSecurity/salary, then you are trying to pad it twice as they are represented by two lines. If thats the case than the iterator is probably over kill and can be rewritten a little cleaner? Currently its a little hard to discern what its trying to do (for someone who hasnt worked on this).

Reply:
That's correct and agreed, I will clean this up a bit! Thanks for the feedback!

Reply:
@SMLuthi Could you take another peek and see if it makes more sense now?